spec/motion-spec/mac_bacon_spec.rb

```diff
@@ -1,23 +1,22 @@
 # -*- encoding : utf-8 -*-
-motion_require '../spec_helper'
 
 class MockObservable
   attr_accessor :an_attribute
 end
 
-describe "NSRunloop aware Bacon" do
+describe 'NSRunloop aware Bacon' do
   describe "concerning `wait' with a fixed time" do
-    it "allows the user to postpone execution of a block for n seconds, which will halt any further execution of specs" do
+    it 'allows the user to postpone execution of a block for n seconds, which will halt any further execution of specs' do
       started_at_1 = started_at_2 = started_at_3 = Time.now
-      #number_of_specs_before = Motion::Spec::Counter[:specifications]
+      # number_of_specs_before = MotionSpec::Counter[:specifications]
 
-      wait 0.5 { (Time.now - started_at_1).should.be.close(0.5, 0.5) }
+      wait(0.5) { (Time.now - started_at_1).should.be.close(0.5, 0.5) }
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.